src/hooks/useIsIframe.ts

```diff
@@ -0,0 +1,3 @@
+export function useIsIframe() {
+  return window !== window.parent
```

Comment:
you don't really need a hook, you can just `export const IS_WITHIN_IFRAME = window && window !== window.parent`